### ISSUE TITLE
Does it support OCR?

### ISSUE THREAD

**Issue by shuaixinjie** [microfocus-idol/idol-oem-tutorials#9]
If it is supported, how should I operate it?

**Comment by brian-cowe-mf** [comment]
Yes OCR supported.  If you have a it enabled in your licence, OCR happens by default when you filter an image; there's no change needed to your code.
